### ISSUE TITLE
Ontology interface doesn't return expected results if the entities have no prefix

### ISSUE THREAD

**Issue by Jasmine8596** [b-it-bots/mas_knowledge_base#8]
Some of the functions in the ontology query interface don't seem to work when the names of the ontology entities (classes, relations) have:
* a URL in the name (e.g. `http://www.semanticweb.org/jasmine/ontologies/2019/8/recipe_fault_diagnosis.owl#State`)
* no prefix in the name (e.g. the name is just `State`)

This particularly holds for the functions `get_subjects_of`/`get_objects_of`, as well as for `get_parent_classes_of`/`get_subclasses of`/`is_instance_of`.

In other words, it seems like the interface only works properly when the ontology entities are defined as `prefix:entity`. This is inconvenient since editors such as Protege save the entities with a full URL by default.

**Comment by alex-mitrevski** [comment]
While this is true, I currently don't see an elegant way of handling different cases in which entities can be stored. I would thus refrain from supporting URLs in the interface for the time being.

The problem with the URL can be circumvented by for instance defining a prefix with a declaration

```
xmlns:prefix="http://www.semanticweb.org/jasmine/ontologies/2019/8/recipe_fault_diagnosis.owl#"
```

and then using `prefix:entity` whenever needed in the ontology.

This is for example resolving most of the issues that our ROPOD ontology has with the interface.

I will address the "no prefix" issue in a PR.
